.github/workflows/cargo_machete.yml

```
@@ -15,7 +15,8 @@ jobs:
         with:
           toolchain: 1.85
       - name: Machete install
-        run: cargo install cargo-machete --locked
+        ## The official cargo-machete action
+        uses: bnjbvr/cargo-machete
```

Apparently GitHub actions ignores any workflow that points to a non-existing tool???

```suggestion
        uses: bnjbvr/cargo-machete@main
```